### DISCUSSION TITLE
Server Install

### DISCUSSION THREAD

**Discussion by colinaamot** [ICIJ/datashare#1225] [General]
What's the best way to deploy this via server mode? Is Docker the only deployment configuration?

**Comment by pirhoo** [comment]
Hello @colinaamot, Docker is recommended because it works the same on every OS but if you're on a Linux server you can either use the `.deb` package or even execute the jar directly:

https://github.com/ICIJ/datashare-installer/releases/tag/13.2.6

We have been using the deb since several years on server, it's probably the lightest method to install Datashare.
